internal/npm_install/npm_install.bzl

```diff
@@ -155,17 +188,26 @@ def _add_scripts(repository_ctx):
         repository_ctx.path(Label("//internal/npm_install:pre_process_package_json.js")),
         {},
     )
-
-    repository_ctx.template(
+    
+    repository_ctx.template( 
```

Comment:
Spurious change, revert (it might require editing the file in an editor that doesn't strip trailing whitespace from lines).